src/main/java/core/basesyntax/figures/Special.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figures;
+
+public interface Special {
+    boolean hasLeg();
+}
```

Comment:
let's rename this interface to `AreaCalculator` and rename its method to `getArea()` and let `Figure` implement this interface and all subclasses must implement this method inside of them

Reply:
Check now pls